doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -192,20 +192,25 @@ If you experience any problems during the process, restart the Programmer app by
 
 .. _nrf9160_gs_connecting_dk_to_cloud:
 
-Connecting the DK to nRF Cloud
-******************************
+.. |DK| replace:: nRF9160 DK
```

Comment:
You need to switch the order of the replace definition and the reference tag, it's currently breaking the build.